### PR TITLE
Account banking not required

### PR DESCRIPTION
**Pull request by gonxi**
Add a check in payment mode for do the account bank optional, for example... manual mode.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rebase as there has been a merge with important changes in these set.

**Comment by gonxi** [comment]
I think that now is OK

File: account_banking_payment_export/models/payment_mode.py [hunk 1]
```diff
@@ -104,3 +104,7 @@ def type_on_change(self):
                 aj_ids = ajo.search([
                     ('type', 'in', ('sale_refund', 'sale'))]).ids
             self.default_journal_ids = [(6, 0, aj_ids)]
+
+    bank_id = fields.Many2one(required=False)
+    check_required = fields.Boolean(default=True, string="Bank account required",
```
Comment (pedrobaeza) [inline account_banking_payment_export/models/payment_mode.py:109]: You should include the check_required at payment type level, not at payment mode. This way, "Manual Direct Debit" and "Manual Bank Payment" have this check unmarked, and SEPA payment types have it checked. The default value for the field should be True to preseve the same behaviour.

**Comment by StefanRijnhart** [comment]
Agreed with @pedrobaeza. Make this a field on the payment mode type as well, and make the field on payment mode an invisible, related field to the first one (set it in an on-change on the payment mode type).

**Comment by sbidoul** [comment]
@gonxi since this one has not moved since one year, I close it. Feel free to reopen if you continue this work.

**Pull request closed without merging**